trin-bridge/src/cli.rs

```diff
@@ -59,11 +59,15 @@ fn check_node_count(val: &str) -> Result<u8, String> {
 /// - Latest: tracks the latest header
 /// - Backfill: starts at block 0
 /// - StartFromEpoch: starts at the given epoch
+///   - ex: "e123" starts at epoch 123
+/// - Single: executes a single block
+///   - ex: "b123" executes block 123
```

Comment:
Nit: In the future something like just `--single-block 123` and `--epoch 123` (that can't be used with `--latest` or `--backfill`) would be better CLI design.

Reply:
agreed! tracking in #783